core/src/test/java/org/apache/druid/timeline/partition/PartitionBoundariesTest.java

```diff
@@ -75,8 +76,23 @@ public void handlesRepeatedValue()
   }
 
   @Test
-  public void serializesDeserializes()
+  public void serializesDeserializes() throws JsonProcessingException
   {
-    TestHelper.testSerializesDeserializes(TestHelper.JSON_MAPPER, target);
+    final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Was there an issue with using `TestHelper`?

Reply:
`TestHelper` is in `druid-processing` while this class is in `druid-core`.